packages/backend/src/assets/ai.json

```
@@ -86,6 +86,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-nodejs",
```

```suggestion
      "id": "chatbot-nodejs-react",
```

as we are now usin the pattern feat-language-framemork now